### PR TITLE
Add an entry for the Canadian Football League API.

### PR DESCRIPTION
**Pull request by yllus**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -691,6 +691,7 @@ API | Description | Auth | HTTPS | CORS |
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [BikeWise](https://www.bikewise.org/documentation/api_v2) | Bikewise is a place to learn about and report bike crashes, hazards and thefts | No | Yes | Unknown |
+| [Canadian Football League (CFL)](http://api.cfl.ca/) | Official JSON API providing real-time league, team and player statistics about the CFL | `apiKey` | Yes | No |
```
Comment (davemachado) [inline README.md]: Please remove "JSON API" from the description.

Comment (yllus) [reply README.md]: With lots of APIs listed being an unofficial source, I thought it'd be helpful to note that this one is direct from the league. "Official providing real-time league..." doesn't make sense as a sentence?

Comment (davemachado) [reply README.md]: Sure, thanks for clarifying.

**Comment by davemachado** [comment]
Thanks!

**Pull request merged**